tests/neptune/internal/channels/test_channels_values_sender.py

```diff
@@ -220,7 +220,7 @@ def test_send_when_waiting_for_next_value_timed_out(self):
             channels_values_sender.send(self._NUMERIC_CHANNEL.name, self._NUMERIC_CHANNEL.channelType, channel_value)
 
         # and
-        time.sleep(self.__TIMEOUT * 2)
+        time.sleep(self.__TIMEOUT * 10)
```

Comment:
Arbitrarly increasing timeout doesn't solve this issue in general, just hides the problem. Find the real reason why more sleep is required & fix instead of this